templates/telco/index.html

```diff
@@ -20,7 +20,7 @@ <h1>
       Scale and complexity brings similar challenges, big or small. Hybrid cloud, public cloud, on-prem only large datacenter or highly distributed infrastructure - Canonical Open Source technology building blocks provide a unified approach to enabling Service Providers to meet any current or future use-cases, from OpenRAN, next generation Core (5G and beyond) or AI at the edge.
       </p>
       <p>
-        <a href="/telco/contact-us" class="p-button--positive js-invoke-modal">Contact us</a>
```

Comment:
I think we still leave this for people who don't have JS enabled, so that it still redirects the user to a generic modal. Also it's not semantic HTML to skip href.

Reply:
Good suggestion. I checked the other branches and they all still have it, so it should fix itself upon merge.